tecton_client/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.0b4"
+__version__ = "0.2.0"
```

Comment:
sooo we've only ever released beta versions, which means even if we were to release `0.2.0b1`, it would still automatically get picked up by users. My plan is to do some internal 'beta testing' just installing straight from github and skip the beta version doesn't really have the effect we want.